keps/sig-api-machinery/2340-Consistent-reads-from-cache/README.md

```diff
@@ -207,6 +216,13 @@ Since falling back to etcd won't work, we should fail the requests and rely on
 rate limiting to prevent cascading failure.  I.e. `Retry-After` HTTP header (for
 well behaved clients) and [Priority and Fairness](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190228-priority-and-fairness.md).
 
+For such situations we will provide a dedicated metric to detect a problem,
+and a per-request override for debugging. If there is a ongoing problem,
+user can reconfigure kube-apiserver to always pass request to etcd by 
+disabling the feature flag. Metric `apiserver_watch_cache_read_wait`
+will measure wait time experienced by reads for watch cache to become fresh.
+Per request override will be done via a HTTP request header `k8s.io/apiserver/skip-watch-cache`.
+
```

Comment:
I think that summarizing the discussion we had offline why we prefer explicit rather than implicit (behind-the-scenes) fallback would be useful here.

[FTR, my main point were:
1) it gives users full control
2) APF differently estimates the cost of requests from cache vs from etcd, so we would be much more exposed to overloading the control-plane]

Reply:
Done.